standard/classes.md

```diff
@@ -5354,13 +5354,13 @@ Task types can vary in their exact definition, but from the language’s point o
 >
 > *end example*
 
-A task builder type is a class or struct type that corresponds to a specific task type ([§15.15.2](classes.md#15152-task-type-builder-pattern)).
+A task builder type is a class or struct type that corresponds to a specific task type ([§15.15.2](classes.md#15152-task-type-builder-pattern)). The task builder type must have the same declared accessibility as its corresponding task type.
```

Comment:
Is this actually true? That would feel like a pity - I could imagine a public task type with an internal task builder type, where basically there can be public methods returning the task type, but other assemblies couldn't create instances of the task type. (I haven't tested this.)